controllers/azuremachine_reconciler.go

```diff
@@ -343,6 +319,7 @@ func (r *azureMachineReconciler) createVirtualMachine(nicName string) (*compute.
 			Size:       r.machineScope.AzureMachine.Spec.VMSize,
 			OSDisk:     r.machineScope.AzureMachine.Spec.OSDisk,
 			Image:      r.machineScope.AzureMachine.Spec.Image,
+			CustomData: *r.machineScope.Machine.Spec.Bootstrap.Data,
 			// Zone:       vmZone,  // TODO: figure out if how to re-enable this
```

Comment:
Let's remove these and open issues about it

Reply:
Leaving it in for now. I'm going to tackle in the next iteration.